tsl/src/chunk.c

```diff
@@ -259,3 +260,92 @@ chunk_invoke_drop_chunks(Oid relid, Datum older_than, Datum older_than_type)
 
 	return num_results;
 }
+
+/*
+ * drop_chunk_replica:
+ *
+ * This function drops a chunk on a specified data node. It then
+ * removes the metadata about the association of the chunk to this
+ * datanode on the access node.
+ */
+Datum
+drop_chunk_replica(PG_FUNCTION_ARGS)
+{
+	const char *node_name = PG_ARGISNULL(0) ? NULL : NameStr(*PG_GETARG_NAME(0));
+	Oid chunk_id = PG_ARGISNULL(1) ? InvalidOid : PG_GETARG_OID(1);
```

Comment:
I just noticed this line. 
`chunk_id` usually means the chunk id allocated by TimescaleDB. So I suggest to rename it, since it is relational id, e.g.:
```suggestion
	Oid chunk_relid = PG_ARGISNULL(1) ? InvalidOid : PG_GETARG_OID(1);
```

Reply:
Ok, done.